demo/src/index.js

```diff
@@ -15,8 +16,17 @@ window.pillarbox = Pillarbox;
 // Expose the player in the window object
 window.player = player;
 
+// Source examples
+window.sourceExamples = {
+  bipbop: {
+    src: 'https://d2zihajmogu5jn.cloudfront.net/bipbop-advanced/bipbop_16x9_variant.m3u8',
```

Comment:
You should use [official URLs](https://developer.apple.com/streaming/examples/) from Apple.

Reply:
Believe it or not, but I'm quite sure that I got this URL from Apple's website years ago, or maybe mmf. Are Apple's URLs reliable in time or do they change for some reason?